test/unit/model/validation.test.js

```diff
@@ -272,7 +271,7 @@ describe(Support.getTestDialectTeaser('InstanceValidator'), () => {
     });
 
     before(function() {
-      this.stub = sinon.stub(current, 'query').callsFake(() => Promise.resolve([User.build({}), 1]));
+      this.stub = sinon.stub(current, 'query').callsFake(async () => Promise.resolve([User.build({}), 1]));
```

Comment:
Promise.resolve can be removed

Reply:
Whoops, will do, thanks for pointing this out, that's another little thing I can improve about my codemod

Reply:
In fact, you can use `.resolves` for all `callsFake` occurences that return promises.